app/service.py

```diff
@@ -21,6 +22,7 @@ def get_activity_sum(user_id: int, activity_name: str) -> int:
     return total
 
 def get_customized_message(activity: Activity = None) -> str: # "activity" is an optional parameter
+    inp = random.randint(0,7)
```

Comment:
The code would be more maintainable if we'd move this line under the definition of "messages" list and if we would use the length of the "messages" list instead of magic number 7